enterprise/tpa_pipeline.py

```diff
@@ -8,36 +8,53 @@
 from django.utils.translation import ugettext as _
 
 from enterprise.models import EnterpriseCustomer, EnterpriseCustomerUser, UserDataSharingConsentAudit
-from enterprise.utils import NotConnectedToEdX
+from enterprise.utils import NotConnectedToOpenEdx
 
 try:
-    from social.pipeline.partial import partial
+    from social_core.pipeline.partial import partial
 except ImportError:
     from enterprise.utils import null_decorator as partial  # pylint:disable=ungrouped-imports
 
 try:
+    from third_party_auth.pipeline import get as get_pipeline_partial
+except ImportError:
+    get_pipeline_partial = None
+
+try:
     from third_party_auth.provider import Registry
 except ImportError:
     Registry = None
 
 
+def verify_third_party_auth_dependencies():
```

Comment:
Would this app ever be installed without social_core or third_party_auth? If not, this check should go away and both requirements should be explicitly called out in setup.py.